app/scripts/modules/core/delivery/create/CreatePipelineModal.tsx

```diff
@@ -200,22 +200,26 @@ export class CreatePipelineModal extends React.Component<ICreatePipelineModalPro
                   <b>Type</b>
                 </div>
                 <div className="col-md-7">
-                  <Select options={[{label: 'Pipeline', value: false}, {label: 'Strategy', value: true}]}
-                          clearable={false}
-                          value={this.state.command.strategy ? {label: 'Strategy'} : {label: 'Pipeline'}}
-                          onChange={this.handleTypeChange}/>
+                  <Select
+                    options={[{label: 'Pipeline', value: false}, {label: 'Strategy', value: true}]}
+                    clearable={false}
+                    value={this.state.command.strategy ? {label: 'Strategy'} : {label: 'Pipeline'}}
+                    onChange={this.handleTypeChange}
```

Comment:
We've been putting the first attribute on the same line as the tag, then indenting subsequent attributes. Is this change driven by lint rules or more conventional in some way?